python-aexpect.spec

```diff
@@ -17,29 +15,27 @@
 %endif
 %global shortcommit	%(c=%{commit};echo ${c:0:7})
 %global gitrel		.%{commit_date}git%{shortcommit}
-%global gittar		%{srcname}-%{shortcommit}.tar.gz
+%global gittar		aexpect-%{shortcommit}.tar.gz
 %endif
 
 # Selftests are provided but skipped because they use unsupported tooling.
 %global with_tests 0
 
-Name: python-%{srcname}
+Name: python-aexpect
 Version: 1.6.2
-Release: 1%{?gitrel}%{?dist}
+Release: 2%{?gitrel}%{?dist}
 Summary: Aexpect is a python library to control interactive applications
-Group: Development/Tools
 
-License: GPLv2
+License: GPLv2+
 URL: https://github.com/avocado-framework/aexpect
 
 %if 0%{?rel_build}
-Source0: https://github.com/avocado-framework/%{srcname}/archive/%{version}.tar.gz#/%{gittar}
+Source0: %{url}/archive/%{version}/%{gittar}
```

Comment:
Interesting, are the variables case insensitive? (I can only see `URL` being defined, not `url`)

Reply:
The tag names (eg., `URL`, `Name`, `Version`, etc.) _are_ case-insensitive, but macro names are case-sensitive. The built-in `%{url}` macro is indeed lower case, and it takes its value from whatever case you happen to use for `URL`.